### DISCUSSION TITLE
Coral Mini-PCIe

### DISCUSSION THREAD

**Discussion by ryanbmcd** [blakeblackshear/frigate#900] [Ask A Question] [answered]
Hey all,
So, I thought I was being smart and found one of the mini PCIe Coral devices for a pretty good price and jumped on it.  Upon arriving, I realized I'm not very hardware-literate and thought the small PCI slot for an additional NIC was the same as a "mini", but it apparently is not.  Does anyone know if there's any kind of adapter or way to interface this mini-PCIe Coral into either a full length PCIe slot or into maybe a spare USB3?  I'm guessing no, but wanted to make sure before cutting my losses.
Thanks!

**Comment by pdecat** [comment, marked as answer]
Hi, there's a Mini PCI-E to PCI-E adapter listed under https://github.com/google-coral/edgetpu/issues/256

**Reply by ryanbmcd** [reply]
You are my hero!  Ordered.  Will try to remember to update this thread with results.

**Reply by ryanbmcd** [reply]
Update: the adapter has been received and can confirm it works.  Thanks again!
